app/components/Views/ImportFromSeed/index.js

```diff
@@ -242,16 +233,22 @@ class ImportFromSeed extends PureComponent {
 	};
 
 	async componentDidMount() {
+		const authType = await Authentication.getType();
+		const previouslyDisabled = await AsyncStorage.getItem(BIOMETRY_CHOICE_DISABLED);
+		const passcodePreviouslyDisabled = await AsyncStorage.getItem(PASSCODE_DISABLED);
+		if (authType.type === AUTHENTICATION_TYPE.BIOMETRIC)
+			this.setState({
+				biometryType: authType.type,
+				biometryChoice: !(previouslyDisabled && previouslyDisabled === TRUE),
+			});
+		else if (authType.type === AUTHENTICATION_TYPE.PASSCODE)
+			this.setState({
+				biometryType: Device.isIos() ? authType.type + '_ios' : authType.type + '_android',
+				biometryChoice: !(passcodePreviouslyDisabled && passcodePreviouslyDisabled === TRUE),
+			});
```

Comment:
instead of using `authType.type` why not destructure here?
```suggestion
		const { type } = await Authentication.getType();
		const previouslyDisabled = await AsyncStorage.getItem(BIOMETRY_CHOICE_DISABLED);
		const passcodePreviouslyDisabled = await AsyncStorage.getItem(PASSCODE_DISABLED);
		if (type === AUTHENTICATION_TYPE.BIOMETRIC)
			this.setState({
				biometryType: type,
				biometryChoice: !(previouslyDisabled && previouslyDisabled === TRUE),
			});
		else if (type === AUTHENTICATION_TYPE.PASSCODE)
			this.setState({
				biometryType: Device.isIos() ? type + '_ios' : type + '_android',
				biometryChoice: !(passcodePreviouslyDisabled && passcodePreviouslyDisabled === TRUE),
			});
```